pyomo/common/errors.py

```diff
@@ -46,3 +47,27 @@ class InfeasibleConstraintException(PyomoException):
 class NondifferentiableError(PyomoException, ValueError):
     """A Pyomo-specific ValueError raised for non-differentiable expressions"""
     pass
+
+
+class WindowsError_def(Exception):
+    """
+    An exception used there is an error configuring a package.
+    """
+
+    def __init__(self, *args, **kargs):
+        Exception.__init__(self, *args, **kargs)  #pragma:nocover
+
+
+if (sys.platform[0:3] != "win"):
+    WindowsError = WindowsError_def
+else:
+    WindowsError = WindowsError
```

Comment:
I think we can remove this and update the two places in Pyomo that were catching `WindowsError` to just catch `OSError` (on Windows, the Python `WindowsError` is a subclass of `OSError`).